react/Field/styles.styl

```diff
@@ -10,10 +10,16 @@
 .o-side
     position absolute
     top 0
```

Comment:
`top 0` and `left 0` are default values, so I don't think we need to precise them.

Reply:
That's actually not true. :)
Default `top` position value is the one the element has in the flow, while the `left` one depends on the direction set in the page (sure we use ltr, but still that's not always true)
https://codepen.io/GoOz/pen/Jxogax

my 2 cents.